<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperatura</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/new/prefix1/recursos/alta-temperatura.png</normaloff>:/new/prefix1/recursos/alta-temperatura.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Centigrados</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLCDNumber" name="outCent">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="inCent">
           <property name="minimum">
            <number>-273</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLCDNumber" name="outFahr">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 0, 127);</string>
           </property>
           <property name="value" stdset="0">
            <double>32.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="inFahr">
           <property name="minimum">
            <number>-459</number>
           </property>
           <property name="maximum">
            <number>210</number>
           </property>
           <property name="value">
            <number>32</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Kelvin</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLCDNumber" name="outKelvin">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 170, 0);</string>
           </property>
           <property name="value" stdset="0">
            <double>273.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="inKelvin">
           <property name="maximum">
            <number>372</number>
           </property>
           <property name="value">
            <number>273</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inKelvin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKelvin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
